platform/ios/src/MGLMapView.mm

```diff
@@ -1622,6 +1626,9 @@ - (void)handlePinchGesture:(UIPinchGestureRecognizer *)pinch
     {
         self.scale = powf(2, [self zoomLevel]);
 
+        if (abs(pinch.velocity) > abs(self.rotate.velocity)) {
```

Comment:
Still concerned that this is comparing properties with different meanings. If it's working, then we can consider as tail-work.

Reply:
Re: https://github.com/mapbox/mapbox-gl-native/pull/14929#discussion_r315429756 

We should only `return` if the minimum number of degrees to rotate has not been met and a zoom triggered by a pinch gesture is occurring. Just checking whether the map is zooming doesn't seem sufficient.